scripts/utils.ts

```diff
@@ -13,14 +13,15 @@ export const ncpP: (src: string, dest: string) => Promise<void> = pify(ncp);
 export const statP: (path: string) => Promise<Stats> = pify(stat);
 export const readFileP: (path: string, encoding?: string) => Promise<Buffer|string> = pify(readFile);
 
-export async function existsP(path: string): Promise<boolean> {
-  let exists = true;
-  try {
-    await statP(path);
-  } catch (e) {
-    exists = false;
-  }
-  return exists;
+export function nodule(nodule: string) {
+  return path.relative(BUILD_DIRECTORY, `node_modules/${nodule}`);
+}
+
+export function existsP(path: string): Promise<boolean> {
```

Comment:
No need to change this further but I'm curious if `statP(path) as Promise<boolean>` would also work? Seems unfortunate to create additional promises just for type conversion.

Reply:
I'm not sure that would work -- we're specifically turning a rejection into a resolution with a `false` value.

Reply:
Ah yes, that makes sense.